packages/firestore/test/integration/api/query.test.ts

```diff
@@ -2193,12 +2186,18 @@ apiDescribe('Queries', (persistence: boolean) => {
       return 'passed';
     };
 
+    // Use LRU memory cache so that the resume token will not be deleted between
+    // calls to `getDocs()`.
+    const localCache = persistence
+      ? persistentLocalCache()
+      : memoryLocalCache({ garbageCollector: memoryLruGarbageCollector() });
+
```

Comment:
While running locally against Prod, it is not using bloom filter:
<img width="1625" alt="Screenshot 2023-06-28 at 3 13 53 PM" src="https://github.com/firebase/firebase-js-sdk/assets/107142260/42c4467c-b8b7-4fe6-9ef2-c753cab9dadf">